src/pyhf/constraints.py

```diff
@@ -125,62 +151,82 @@ def _precompute(self):
             try:
                 poisson_constraint_rate_factors.append(parset.factors)
             except AttributeError:
-                poisson_constraint_rate_factors.append(
-                    default_backend.shape(self.par_indices[parslice])
-                )
+                # this seems to be dead code
+                # TODO: add coverage
```

Comment:
@lukasheinrich can you comment more on the state of this? Or is this done and need to get removed?

Reply:
The refactor to paramviewer does not touch this code so I chose not to fix it, but I noticed that it's dead code which we should review how that ended up in there and possibly remove it in a separate PR

Reply:
Is there an issue for this? If not, can you make one and note the issue number in the comment?

Reply:
done